<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCocosCreaterClass</class>
 <widget class="QDialog" name="CCocosCreaterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cocos2d-x工程创建器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbProjectName">
       <property name="text">
        <string>工程名称：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leProjectName">
       <property name="text">
        <string>NewProject</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbPackageName">
       <property name="text">
        <string>包名称：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lePackageName">
       <property name="text">
        <string>NewPacket</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="RbtnCpp">
         <property name="text">
          <string>Cpp</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RbtnLua">
         <property name="text">
          <string>Lua</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RbtnJs">
         <property name="text">
          <string>Js</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbProjectPath">
       <property name="text">
        <string>路径：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leProjectPath"/>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnBrowser">
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="text">
        <string>创建</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cocoscreater.qrc"/>
 </resources>
 <connections/>
</ui>
